src/bench/pqs/linden.cpp

```diff
@@ -35,16 +36,38 @@ Linden::~Linden()
 
 void
 Linden::insert(const uint32_t &key,
-               const uint32_t & /* Unused */)
+               const uint32_t &value)
 {
-    linden_insert(m_q->pq, key);
+    /*Add 1 to key to support 0 keys*/
+    linden_insert(m_q->pq, key+1, value);
+}
+
+void
+Linden::insert(const size_t &key,
```

Comment:
I think we could make sure that call sites use uint32_t but that would require casting from size_t or other changes so I don't think it is a good idea. Do you have any particular reason for wanting to avoid the overloads?